_schemaV2.graphql

```diff
@@ -1033,29 +1033,15 @@ type ArtistTargetSupplyMicrofunnel {
 
     # Randomize the order of artworks for display purposes.
     randomize: Boolean
-
-    # Number of artworks to return
-    size: Int
   ): ArtworkConnection
-  metadata: ArtistTargetSupplyMicrofunnelMetadata
+  metadata: TargetSupplyMicrofunnelMetadata
```

Comment:
I _think_ it's okay for me to change the name of this type, because the underlying _shape_ is the same. I opted for this new name because `Artist` doesn't apply to the root field.